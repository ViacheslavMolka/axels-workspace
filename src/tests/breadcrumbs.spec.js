import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Breadcrumbs from '../components/breadcrumbs';


configure({ adapter: new Adapter() });

it('Breadcrumbs snapshot', () => {
    const component = shallow(<Breadcrumbs />);
    expect(component).toMatchSnapshot();
});