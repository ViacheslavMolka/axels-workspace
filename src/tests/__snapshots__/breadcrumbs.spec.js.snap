// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs snapshot 1`] = `ShallowWrapper {}`;
